Firmware/backlight.h

```diff
@@ -0,0 +1,32 @@
+//backlight.h
+#ifndef _BACKLIGHT_H
+#define _BACKLIGHT_H
+
+#include <inttypes.h>
+#include "Marlin.h"
+#include "pins.h"
+
+enum Backlight_Mode
```

Comment:
we can probably save a few bytes making this an enum class of type uint8_t
the compiler won't expect these numbers being 16bit integers then